pkg/known/labels.go

```diff
@@ -24,6 +24,8 @@ const (
 	ClusterBootstrappingLabel = "clusters.clusternet.io/bootstrapping"
 
 	ObjectCreatedByLabel = "clusternet.io/created-by"
+	ObjectControlledByNsLabel = "clusternet.io/controlled-by-ns"
+	ObjectControlledByDescLabel = "clusternet.io/controlled-by-desc"
```

Comment:
I did not get the meanings and difference of L27 and L28.

Reply:
label the resouce with Description namespace and name seperately.